assets/icons.tsx

```diff
@@ -853,3 +853,27 @@ export const Back: FC<IIconProps> = ({ width = 24, height = 24, color = '#007AFF
     <path d='M15 6L9 12L15 18' stroke={color} strokeWidth='1.6' />
   </svg>
 );
+
+export const Copy: FC<IIconProps> = ({ width = 24, height = 24, color = '#003D80' }) => (
+  <svg
+    xmlns='http://www.w3.org/2000/svg'
+    width={width}
+    height={height}
+    viewBox='0 0 24 24'
+    fill='none'
+  >
+    <path
+      d='M6.00044 6.12787L6 15.75C6 18.75 6.75 19.5 9.50344 19.492L9.75 19.5L16.3716 19.5011C16.0624 20.3744 15.2293 21 14.25 21H9C6 21 4.5 19.492 4.5 16.5V8.25C4.5 7.27017 5.12631 6.43663 6.00044 6.12787ZM17.25 3C18.4926 3 19.5 4.00736 19.5 5.25V15.75C19.5 16.9926 18.4926 18 17.25 18H9.75C8.25 18 7.5 17.25 7.5 15.75V5.25C7.5 4.00736 8.50736 3 9.75 3H17.25ZM17.25 4.5H9.75C9.33579 4.5 9 4.83579 9 5.25V15.75C9 16.1642 9.33579 16.5 9.75 16.5H17.25C17.6642 16.5 18 16.1642 18 15.75V5.25C18 4.83579 17.6642 4.5 17.25 4.5Z'
+      fill={color}
+    />
+  </svg>
+);
+
+export const Tick: FC<IIconProps> = ({ width = 24, height = 24, color = '#007AFF' }) => (
```

Comment:
svg o'rniga html character code ishlatib ko'ring

Reply:
bu qolib ketibdi

Reply:
uni qanday qilinadi oldin qilmagan edim?

Reply:
&#10003;  shuni html ichida render qilsez bo'ladi. Bu ham &nbsp; kabi ishlaydi. Lekin bizni dizaynga to'g'ri keladigani yo'q ekan